src/zenml/zen_stores/schemas/step_run_schemas.py

```diff
@@ -310,9 +310,11 @@ class StepRunInputArtifactSchema(SQLModel, table=True):
         nullable=False,
         primary_key=True,
     )
+    # Note: We keep the name artifact_id instead of artifact_version_id here to
```

Comment:
Even though this is a painful process, we basically did this for the `run_metadata` column right? I would recommend doing this if possible for the StepRunInput- and OutputArtifactSchema as well. Otherwise, it might be really confusing in the future, as Artifacts have their own table.

Reply:
Actually, scratch this. There is an upcoming PR that will completely change the way foreign keys work at this table. Let's not change the name here.

Reply:
I don't think we've done anything like this before. `artifact_id` is part of the primary key of `step_run_input_artifact` and `step_run_output_artifact` and there's no way to rename a primary key constraint, so we would need to:
- manually create an exact copy of the table with renamed `artifact_version_id` field and primary key constraint
- migrate all data from the old table
- drop the old table
- rename the new table to the old table

As you can imagine, this would be extremely error-prone so I'd rather not risk it unless there is a good reason for it.